examples/university-java-springboot/pom.xml

```diff
@@ -64,6 +65,10 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-webflux</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-webmvc</artifactId>
```

Comment:
How did adding the `webmvc` dependency and setting a `webmvc` profile help you solve the issue? Might be helpful to add that to the PRs description for future investigations into this area.